sdks/python/apache_beam/runners/dataflow/internal/apiclient_test.py

```diff
@@ -173,6 +173,28 @@ def test_harness_override_absent_in_unreleased_sdk(self):
       env = apiclient.Environment([], pipeline_options, '2.2.0')
       self.assertNotIn(override, env.proto.experiments)
 
+  def test_labels(self):
+    pipeline_options = PipelineOptions(
+        ['--project', 'test_project', '--job_name', 'test_job_name',
+         '--temp_location', 'gs://test-location/temp'])
+    job = apiclient.Job(pipeline_options)
+    self.assertIsNone(job.proto.labels)
+
+    pipeline_options = PipelineOptions(
+        ['--project', 'test_project', '--job_name', 'test_job_name',
+         '--temp_location', 'gs://test-location/temp',
+         '--label', 'key1=value1',
+         '--label', 'key2',
+         '--label', 'key3=value3'])
```

Comment:
<!--new_thread; commit:d585b378c36673ccdecbdb9b3c7dcf131ed463df; resolved:0-->
Can we test the "--labels" alias too?